src/Microsoft.VisualStudio.Composition.Analyzers/AnalyzerReleases.Unshipped.md

```diff
@@ -5,3 +5,7 @@
 Rule ID | Category | Severity | Notes
 --------|----------|----------|-------
 VSMEF003 | Usage | Warning | Exported type not implemented by exporting class
+VSMEF004 | Usage | Error | VSMEF004ExportWithoutImportingConstructor
+VSMEF005 | Usage | Error | VSMEF005MultipleImportingConstructors
+VSMEF006 | Usage | Warning | VSMEF006ImportNullability
+VSMEF007 | Usage | Warning | VSMEF007DuplicateImportAnalyzer
```

Comment:
Can we replace the values for the Notes column with something like what you have in index.md?
Also, I like your choice of severities here.